src/main/java/backend/resource/TurboIssue.java

```diff
@@ -148,6 +153,18 @@ public TurboIssue(String repoId, Issue issue) {
         this.metadata = IssueMetadata.empty();
         this.repoId = repoId;
         this.markedReadAt = Optional.empty();
+        this.relatedIssueOrPRUrl = setrelatedIssueOrPRUrl(issue);
+    }
+
+    private Optional<String> setrelatedIssueOrPRUrl(Issue issue) {
+        if (isPullRequest(issue) && issue.getBodyText() != null) {
+            Pattern p = Pattern.compile("Fixes <a href=\"(.*?)\"", Pattern.DOTALL);
```

Comment:
Do we need DOTALL for this? There shouldn't be a newline in the URL. Is it possible to use `[^"]*` over `.*?`, so it's more explicit?

There are [lots of other strings GitHub accepts](https://help.github.com/articles/closing-issues-via-commit-messages/) to close issues.

The regex portion can probably be put in some GitHub-specific class, and should be tested.